index.py

```diff
@@ -81,8 +83,8 @@ def sort_app(organize_by, app_list):
             organized_app_list['All apps'].append(app)
     else:
         for app in app_list:    
-            if app.get(organize_by) is not None:
-                # check if it already exisits in the organized_app_list dictionary.
+            if len(app.get(organize_by)) > 0:
+                # list of categories/developers/modules associated with the app.
                 items = app.get(organize_by)
```

Comment:
Same as above - best to return a default empty list (or something that will fail as expected when doing a comparison on it)